nano/node/transport/udp.hpp

```diff
@@ -172,7 +172,7 @@ namespace transport
 		attempts;
 		// clang-format on
 		boost::asio::strand<boost::asio::io_context::executor_type> strand;
-		boost::asio::ip::udp::socket socket;
+		std::shared_ptr<boost::asio::ip::udp::socket> socket;
```

Comment:
This could probably be a `unique_ptr` as I don't think anything else shares ownership of it (unless I missed it)

Reply:
Updated